next/src/components/footer/Footer.tsx

```diff
@@ -48,6 +67,24 @@ const Footer: React.FC<{}> = () => {
           B03 CSE Building K17, UNSW
           <br />
           csesoc@csesoc.org.au
+          <br/><br/>
+          <ImagesContainer>
+            <a href="https://discord.gg/AM4GB5zuB6">
```

Comment:
pls change the <a> tags into next/link tags! so import the following `import Link from next/link` and simply just replace the `<a>` tags with `<Link>` tags :>